frontend_vue/src/utils/tokenServices.ts

```diff
@@ -208,7 +208,7 @@ export const tokenServices: TokenServicesType = {
       'Scan this QR Code with the WireGuard app on your phone or copy the config below.',
   },
   [TOKENS_TYPE.AZURE_ENTRA_CONFIG]: {
-    label: 'Azure Entra Config ID',
+    label: 'Azure Entra config ID',
```

Comment:
is this the label we show on the front page? it should be `Azure Entra ID login`

Reply:
I fix it, thank you!